pkgs/development/python-modules/cherrypy/default.nix

```diff
@@ -1,54 +1,39 @@
 { lib, buildPythonPackage, fetchPypi, isPy3k
```

Comment:
```suggestion
{ lib, buildPythonPackage, fetchPypi
```

Reply:
I added `disabled = !isPy3k` just to make sure.